File: app/src/main/java/com/intelegain/agora/model/change_password/ChangePasswordResponse.java
package com.intelegain.agora.model.change_password;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ChangePasswordResponse {
    @SerializedName("Status")
    @Expose
    public String status;
    @SerializedName("Message")
    @Expose
    public String message;
}

File: app/src/main/java/com/intelegain/agora/model/change_password/ChangePasswordResponse.kt
package com.intelegain.agora.model.change_password

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ChangePasswordResponse {
    @SerializedName("Status")
    @Expose
    var status: String? = null
    @SerializedName("Message")
    @Expose
    var message: String? = null
}